### ISSUE TITLE
More 

### ISSUE THREAD

**Issue by IndictionEve** [pyfa-org/Pyfa#813]
I feel like pyfa hide some cool and useful informations and features.
I would like to make some medium GUI changes:
- [ ] Firepower Panel: Resist pattern gauges for Firepower (like Resistances) to show pattern resistance
- [ ] Firepower Panel: Show damage specific (EM/Thermal/Kinectic/Explosive) dps as tooltips
- [ ] Targeting+Misc Panel: add mass, agility, special cargo holds as separate labels 
- [ ] Targeting+Misc Panel: regroup labels into subgroups (targeting, propulsion, cargo)
- [ ] Capacitor Panel: add "peak recharge rate" and "recharge time" as separate labels

what do you think about it?

**Comment by Ebag333** [comment]
Check out #690

>Firepower Panel: Resist pattern gauges for Firepower (like Resistances) to show pattern resistance

I want to do this very badly, but @blitzmann doesn't want to make the Stats pane take up any more space.

I also want to expose a button users can click for profiles, because hiding that is terrible UI.

>Firepower Panel: Show damage specific (EM/Thermal/Kinectic/Explosive) dps as tooltips

Sounds good.

>Targeting+Misc Panel: add mass, agility, special cargo holds as separate labels
>Targeting+Misc Panel: regroup labels into subgroups (targeting, propulsion, cargo)

I really want to break this out. It's a jumble of stats that don't really belong together, and make not sense to (other than in the "hey, here's a spot to put random stuff" sense).

Also, Ghost Fitting has it's own dedicated drone pane. Why don't we? :)

>Capacitor Panel: add "peak recharge rate" and "recharge time" as separate labels

This is getting a bit of a rework in #812 so we can provide a lot more info.

The amount of info here is pretty sparse, and some of it isn't very accurate.  I'd rather show a genuine peak regen rather than peak regen + modules.  It'd be better if instead of doing +GJ/s and -GJ/s, we did peak regen and then module/projected modifiers.  So if you have a nos running, for example, it gets added under the modifier number (which would take the place of the bottom number) rather than affecting the peak regen.

So for example, say we have a ship with a peak regen of 8.89 GJ/s, and we have -9.49 GJ/s in modules that use cap, we would show:
Peak: 8.89 GJ/s
Used: -9.49 GJ/s

If we add a small nos to that (+4 GJ/s) we see:
Peak: 8.89 GJ/s
Used: -5.49 GJ/s

This to me is a *MUCH* better metric, because throwing positive modules (nos/boosters) into peak regen, because it hides how much regen your ship actually gets (and badly upsets it, because you'll never actually see +4 GJ/s from a nos, because it comes in spurts not every tick).  By comparing module usage to actual ship peak regen, you get a better idea how close you are to cap stability.

Using Gnosis, we can currently report on the follow cap metrics:
1. Peak passive regen delta amount
2. Peak passive regen percent (so what percent you hit maximum, ranges roughly from 23-28 percent)
3. Cap sim low water mark (what % of capacitor you hit where you are considered stable)
4. Cap sim low water mark time (how long it took to hit that mark)

Three of those exist today (though number 1 is hidden as other things modify it).  Number 2 doesn't exist today, and I don't know that we need to expose it (though a tooltip would be nice).

Take a look at those two PRs, and let me know what you think @IndictionEve

**Comment by IndictionEve** [comment]
> I want to do this very badly, but @blitzmann doesn't want to make the Stats pane take up any more space.

But are statistics not what fitting tools are all about? (aka number po*n)? :)
Left a comment in #690. Think it would be a great improvement for visibility.

The cap and shield simulation is exactly what I looking for, wow.

**Comment by Ebag333** [comment]
>But are statistics not what fitting tools are all about?

Yes, but he wants to be able to accommodate the poor saps on 800x600 laptops.

I joke, but 1024x768 isn't unheard of even in this day, and most folk probably don't use Pyfa full screen (so even if they have a 4k display, doesn't mean we get to use 4k of it).

Because wxPython doesn't handle switching displays well (that's a bit of an understatement), if we have separate panes we can't really show/hide them on demand very easily.  Which is why we use the button toggle hack on the firepower pane and don't do it on demand.

It's _super_ common for the top pane to get "stuck" switching between drones and cargo (and displaying the wrong one).  So on demand switching just isn't a great solution.

So we're stuck with being fairly static in our displays....which isn't entirely a bad thing.  Muscle memory and all, we don't want to be moving around _where_ to find info too much.

If you want to submit improvements to that PR, you can submit them against the branch I have on my repo.  I was just talking to @blitzmann about this today, I'd rather the branch be stored in Pyfa-org so someone could just pick it up, but this is what we got for right now.

The last big technical thing left on the stats pane is getting the resist/damage profile context menu working on button click (instead of right click), the catch there is that it's built and attached at the parent level and not the pane level.  Haven't figured out a way around that, and I'm terribad with wxPython.

The last big _non-technical_ thing is getting @blitzmann to merge it.  He hates the look of some of the panes.

**Comment by TheNathanBlack** [comment]
Just getting mass on the panel would be a huge help for us wormholers...

**Comment by vkr-13** [comment]
Agreed, mass is as important as agility as far as wormhole life is concerned.

Are we not able to add a hamburger menu to the targeting & misc group where the user can choose to add additional statistics?
